Monika After Story/game/script-ch30.rpy

```diff
@@ -902,6 +902,8 @@ label ch30_main:
 
 label continue_event:
     m "Now, where was I..."
+    elif:
+        m "Alright, where I stopped?..."
```

Comment:
I don't think we need this variety here in particular, unless there are a ton of people who force close MAS. Even then, I don't think we want to cater too much to that audience.